plugin/build.gradle

```diff
@@ -335,6 +336,10 @@ configurations.all {
     resolutionStrategy.force 'commons-codec:commons-codec:1.15'
     resolutionStrategy.force 'org.slf4j:slf4j-api:1.7.36'
     resolutionStrategy.force 'org.codehaus.plexus:plexus-utils:3.3.0'
+    resolutionStrategy.force 'com.fasterxml.jackson.core:jackson-databind:2.17.0'
+    resolutionStrategy.force 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.17.0'
```

Comment:
can the hardcode 2.17.0 be replaced by `${versions.jackson}`?

Reply:
Removed.